pkgs/games/gscrabble/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, buildPythonApplication, fetchFromGitHub
+, gtk3, gobject-introspection, wrapGAppsHook, gst_all_1
+, python3, gnome3, hicolor-icon-theme }:
+
+with python3.pkgs; buildPythonApplication rec {
```

Comment:
I personally find this `with python3.pkgs` a bit too far up to miss - I'd add it to each `*Inputs` list instead.

Reply:
done